engines/libreoffice/Dockerfile

```diff
@@ -46,4 +46,4 @@ EXPOSE 8090
 
 USER ${LIBREUSERNAME}
 
-ENTRYPOINT java $JAVA_OPTS -jar /usr/bin/${env.project_artifactId}.jar
+ENTRYPOINT ["/bin/sh", "-c", "if [[ \"$(uname -m)\" != \"x86_64\" ]]; then export LIBREOFFICE_HOME=${LIBREOFFICE_HOME:=/usr/lib64/libreoffice}; fi && java $JAVA_OPTS -jar /usr/bin/${env.project_artifactId}.jar"]
```

Comment:
better move to a dedicated script?

not sure why we are using ENTRYPOINT here which is usually useful when you package CLI utilies and you expects user be able to provide additional arguments while using `docker run`. If it's a server application CMD should be enough.